v03_pipeline/lib/tasks/write_new_variants_table.py

```diff
@@ -234,6 +235,15 @@ def create_table(self) -> hl.Table:
             rdc_ht = self.annotation_dependencies[f'{rdc.value}_ht']
             new_variants_ht = new_variants_ht.join(rdc_ht, 'left')
 
+        # Register the new variant alleles to the Clingen Allele Registry
+        # and annotate new_variants table with CAID.
+        if self.dataset_type.should_send_to_allele_registry:
+            for ar_ht in register_alleles_in_chunks(
+                new_variants_ht,
+                self.reference_genome,
+            ):
+                new_variants_ht = new_variants_ht.join(ar_ht, 'left')
```

Comment:
There's nothing in `ar_ht` that's not in `new_variants_ht` right?

Reply:
fwiw I do think the `left` join is correct and safest!  I guess the alternative would be `outer` which probably doesn't make sense.

Reply:
> There's nothing in ar_ht that's not in new_variants_ht right?
that's correct. it's also possible for `ar_ht` is to be missing variants that are in the `new_variants_ht`, hence the `left` join